problem_builder/dashboard.py

```diff
@@ -466,6 +470,26 @@ def add_error(msg):
                         .format(key=key, value=value)
                     )
 
+                if key not in data.mentoring_ids:
+                    add_error(
+                        _(u"Exclude questions is malformed: mentoring url_name {url_name} is not added to Dashboard")
```

Comment:
This reads weirdly in English. Recommend doing, `"'exclude_questions' is malformed"`